lms/djangoapps/commerce/utils.py

```diff
@@ -46,8 +46,7 @@ def __init__(self):
 
     def is_enabled(self, user):
         """ Check if the user is activated, if the service is enabled and that the site is not a microsite. """
-        return (user.is_active and self.config.checkout_on_ecommerce_service and not
-                helpers.is_request_in_themed_site())
+        return user.is_active and self.config.checkout_on_ecommerce_service
```

Comment:
We do have model-backed config for Sites waiting in the wings (WL-327 / #11806) -- we will be picking it up as soon as WL-325 / #11885 clears its gates.